hal-roborio/hal_impl/types.py

```diff
@@ -1,3 +1,5 @@
+# fmt: off
```

Comment:
Surely this doesn't look _that_ terrible after formatting.

Reply:
It's not very good, as it's fairly abnormal code. And I'd rather it stay the way it is.